packages/form/src/components/NumberInputFieldV2/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { NumberInputV2 } from '@ultraviolet/ui'
+import type { ComponentProps, FocusEvent } from 'react'
+import { useController } from 'react-hook-form'
+import type { FieldPath, FieldValues, Path, PathValue } from 'react-hook-form'
+import { useErrors } from '../../providers'
+import type { BaseFieldProps } from '../../types'
+
+type NumberInputV2Props<
+  TFieldValues extends FieldValues,
+  TName extends FieldPath<TFieldValues>,
+> = BaseFieldProps<TFieldValues, TName> &
+  Partial<
+    Pick<
```

Comment:
Not better to use omit instead of that big list ?

Reply:
@DorianMaliszewski Pick was more explicit for this case as we add BaseFieldProps and then we pick from the component.

I will ad name and required are part of BaseFieldProps already but I will add className